ios/Podfile

```diff
@@ -0,0 +1,41 @@
+# Uncomment this line to define a global platform for your project
```

Comment:
Not positive on this file honestly, Android Studio / flutter made it.. maybe junk?